gulpfile.js

```diff
@@ -69,6 +70,37 @@ var webpackConfig = {
   plugins: gulpPlugins
 };
 
+gulp.task('test-unit', function (done) {
+    new KarmaServer({
+        configFile: __dirname + '/karma.conf.js'
+    }, done).start();
+});
+
+gulp.task('test-coverage', function (done) {
+    new KarmaServer({
+        configFile: __dirname + '/karma.conf.js',
+        reporters: ['mocha', 'coverage', 'threshold'],
+        singleRun: true,
+        webpack: {
```

Comment:
I think that a follow up PR which aligns these would be nice! :) For this PR it would be nice to remove the changes to existing source files that simply added the `.jsx` extensions - ie https://github.com/onefinestay/react-daterange-picker/pull/72#discussion-diff-41400016